src/fundus/publishers/de/gamestar.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class GamestarParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div.article-content > p")
```

Comment:
Judging by [this](https://www.gamestar.de/artikel/neu-bei-steam,3412554.html) article, there is a lot of bloat, `captions` etc., extracted alongside the actual content. Further, this newspaper seems to include `subheadlines` (headlines between paragraphs) and a `summary` (little intro text). You can example for both [here](https://www.gamestar.de/artikel/spielerin-baut-skyrim-setup-und-spuert-schmerzen,3412551.html). Could you add those?